components/style/str.rs

```diff
@@ -149,5 +149,5 @@ pub fn str_join<I, T>(strs: I, join: &str) -> String
 /// Returns true if a given string has a given prefix with case-insensitive match.
 pub fn starts_with_ignore_ascii_case(string: &str, prefix: &str) -> bool {
     string.len() > prefix.len() &&
```

Comment:
I guess this should technically be >= (though it doesn't matter due to the callsites). I'll fix too.